Cargo.toml

```diff
@@ -43,7 +43,8 @@ json = "0.11"
 hex = "0.4.3"
 
 [features]
-default = ["std"]
+default = ["std", "jit"]
+jit = []
```

Comment:
This would also need to go into the first commit.

It's still subject to the caveats regarding support for Windows.